tests/integration/components/registries/partial-registration-modal/component-test.ts

```diff
@@ -45,14 +54,21 @@ module('Integration | Component | partial-registration-modal', hooks => {
         const rootNode = await this.store.findRecord('node', root.id);
         this.set('rootNode', rootNode);
         this.set('isOpen', false);
+        const nodes = [];
+        this.set('onContinue', () => {
+            assert.ok(nodes.length = 3);
```

Comment:
Same here. `=` or `===`?

Reply:
Should add an` assert.expect()` as above